setup.cfg

```diff
@@ -39,30 +39,34 @@ console_scripts =
 [options.extras_require]
 dev =
     pytest
-    ruff
-    mypy
-    types-PyYAML
 
+# https://mypy.readthedocs.io/en/stable/config_file.html
 [mypy]
-mypy_path = python
-packages = rhc_playbook_lib, rhc_playbook_verifier, rhc_playbook_signer
```

Comment:
Pre-commit always passes target-files to mypy, and mypy only makes use of the `packages` config option when no files are passed. I felt that leaving this here was misleading.